modules/confidential-data/service_accounts.tf

```
@@ -36,7 +36,7 @@ locals {
 
 module "dataflow_controller_service_account" {
   source       = "terraform-google-modules/service-accounts/google"
-  version      = "~> 3.0"
+  version      = "~> 4.1.1"
```

Lets pin to major versions.
```suggestion
  version      = "~> 4.1"
```